packages/design-system/src/styles/components/_SingleInputDateField.scss

```diff
@@ -36,8 +36,6 @@
 
 /* Hide elements for devices that are not screen readers */
 .rdp-vhidden {
-  -moz-appearance: none;
```

Comment:
Linter doesn't like prefixes. I believe the autoprefixer package is supposed to handle these in the Gulp file, so this removal makes sense